src/cli.ts

```diff
@@ -30,6 +30,7 @@ let options = yargs
     .option('files', { type: 'array', description: 'The list of files (or globs) to include in your project. Be sure to wrap these in double quotes when using globs.' })
     .option('host', { type: 'string', description: 'The host used when deploying to a Roku.' })
     .option('ignore-error-codes', { type: 'array', description: 'A list of error codes that the compiler should NOT emit, even if encountered.' })
+    .option('no-project', { type: 'boolean', defaultDescription: 'false', description: 'when flag set and also project path is null, no bsconfig.json is expected' })
```

Comment:
I really want to call this one `--noproject` for some reason, but `--no-project` correctly follows our other naming conventions. So...let's add an alias to support both. 

```suggestion
    .option('no-project', { type: 'boolean', defaultDescription: 'false', description: 'when flag set and also project path is null, no bsconfig.json is expected', alias: ['noproject', 'noProject'] })
```